### PR TITLE
Fix date ordinalNumber for ja/zh-CN/zh-TW

### PR DESCRIPTION
**Pull request by skyuplam**
The ordinal number for date has suffix in ja, zh-CN and zh-TW.
e.g.
  1st of January in English
  1月1日 in Japanese
  1月1日 in Traditional Chinese
  1月1日 in Simplified Chinese

### CODE REVIEW

File: src/locale/ja/snapshot.md [hunk 1]
```diff
@@ -42,8 +42,8 @@
 |                                 |              | 2019-12-01T12:13:14.015Z | 48                                                   | 2019-11-25T00:00:00.000Z |
 | ISO week of year                | Io           | 2019-01-01T12:13:14.015Z | 1                                                    | 2018-12-31T00:00:00.000Z |
 |                                 |              | 2019-12-01T12:13:14.015Z | 48                                                   | 2019-11-25T00:00:00.000Z |
-| Day of month                    | do           | 2019-02-11T12:13:14.015Z | 11                                                   | 2019-02-11T00:00:00.000Z |
-|                                 |              | 2019-02-28T12:13:14.015Z | 28                                                   | 2019-02-28T00:00:00.000Z |
+| Day of month                    | do           | 2019-02-11T12:13:14.015Z | 11 日                                                | Invalid Date             |
```
Comment (ykzts) [inline src/locale/ja/snapshot.md]: "Invalid Date"?

File: src/locale/zh-CN/_lib/localize/index.js [hunk 2]
```diff
@@ -143,13 +143,8 @@ function ordinalNumber(dirtyNumber, dirtyOptions) {
...
 
-  if (
-    unit === 'date' ||
-    unit === 'hour' ||
-    unit === 'minute' ||
-    unit === 'second'
-  ) {
-    return number.toString()
+  if (unit === 'date') {
+    return number.toString() + '号'
```
Comment (cubicwork) [inline src/locale/zh-CN/_lib/localize/index.js]: 1. According to Chinese National Standard (GB/T 7408-2005), the unit of date should be '日' 
2. Could you explain why you remove code like `... unit === 'hour' || unit === 'minute' || unit === 'second' ... `
3. if put a unit for 'date', unit for  'hour' / 'minute' / 'second' might be necessary

File: src/locale/ja/snapshot.md (hunk 1 above)
Comment (skyuplam) [reply src/locale/ja/snapshot.md]: Fixed.

File: src/locale/zh-CN/_lib/localize/index.js (hunk 2 above)
Comment (skyuplam) [reply src/locale/zh-CN/_lib/localize/index.js]: Fixed.

Comment (cubicwork) [reply src/locale/zh-CN/_lib/localize/index.js]: thx

**Comment by skyuplam** [comment]
This PR is still blocked :(

**Comment by kossnocorp** [comment]
Thank you so much!

**Pull request merged**